packages/fiori/src/ShellBar.hbs

```diff
@@ -32,6 +32,7 @@
 				@click="{{_header.press}}"
 				aria-haspopup="{{menuBtnHasPopup}}"
 				aria-expanded="{{menuPopoverExpanded}}"
+				data-ui5-stable="menuButton"
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** Would it be better not to reference "Button" in this `data-ui5-stable`, since other existing `data-ui5-stable` have values that aren't so specific to the role of element?

```suggestion
				data-ui5-stable="menu"
```